tests/Bridge/Doctrine/Persister/ObjectManagerPersisterTest.php

```diff
@@ -88,10 +93,41 @@ public function testCanPersistAnEntity($entity, bool $exact = false)
         $this->assertEquals(1, count($result));
 
         if ($exact) {
-            $this->assertEquals($entity, $result[0]);
+            $this->assertEquals($originalEntity, $result[0]);
         }
     }
 
+    public function testCanPersistMultipleEntitiesWithExplicitIdentifierSet()
+    {
+        $dummy = new DummyWithIdentifier();
+        $dummy->id = 100;
+        $this->persister->persist($dummy);
+
+        $dummy = new DummyWithIdentifier();
```

Comment:
what is the difference with the previous one? That this time there is two of them?